io/src/ascii_io.cpp

```diff
@@ -45,12 +45,8 @@
 #include <cstdint>
 
 //////////////////////////////////////////////////////////////////////////////
-pcl::ASCIIReader::ASCIIReader ()
+pcl::ASCIIReader::ASCIIReader (): sep_chars_(", \n\r\t"), extension_(".txt"), name_("AsciiReader")
```

Comment:
Should we prefer field initialization instead of constructor initialization?

Reply:
I believe field initialization doesn't have any advantage over initialization in the constructor (performance or otherwise), does it?

Reply:
Nothing but it can simplify the constructors, more apparent if one have many: https://stackoverflow.com/questions/35582045/whats-the-reason-for-initializing-fields-inside-class
And its suggested by the core guidelines: could have a quick look at this https://www.youtube.com/watch?v=O65lEiYkkbc, the first 18 mins 😄

Reply:
A bit easier to read also (personal opinion).

Reply:
I switched to in-class initialization in a few cases where it made the most sense (especially in image_grabber.cpp)